Marlin/src/module/probe.cpp

```diff
@@ -823,7 +816,7 @@ float Probe::run_z_probe(const bool sanity_check/*=true*/) {
           #if EXTRA_PROBING > 0
             < TOTAL_PROBING - 1
           #endif
-        ) do_z_clearance(Z_CLEARANCE_MULTI_PROBE);
+        ) do_z_clearance(z + (Z_CLEARANCE_MULTI_PROBE));
```

Comment:
I brought `z` back here because it includes `largest_sensorless_adj` and that was part of the raise here. If we want to use `Z_CLEARANCE_MULTI_PROBE` as a relative raise, then we should also add `largest_sensorless_adj`, but `z` already includes that.

Reply:
This makes `Z_CLEARANCE_MULTI_PROBE` a relative move, whilst others will be absolute. Worth bearing in mind, but I don't think it would be problematic at all. Perhaps renaming to `Z_LIFT_MULTI_PROBE` would be appropriate.